### PR TITLE
Sort requests by priority

### PR DESCRIPTION
**Pull request by VasurTrekkson**
Sort Requests after Priority

### CODE REVIEW

File: src/main/java/com/lothrazar/storagenetwork/util/RequestBatch.java [hunk 1]
```diff
@@ -23,4 +24,40 @@ public void extractStacks(IConnectableLink providerStorage, Integer slot, Item i
...
+    public void sort() {
+        Collection<List<Request>> requests = this.values();
+        for(List<Request> requestList : requests) {
+            quickSort(requestList, 0, requestList.size() - 1);
+        }
+    }
+
+    private void quickSort(List<Request> requestList, int start, int end) {
+        if (start < end) {
+            int partitionInd = partition(requestList, start, end);
```
Comment (IIpragmaII) [inline src/main/java/com/lothrazar/storagenetwork/util/RequestBatch.java:37]:
```suggestion
            int partitionInt = partition(requestList, start, end);
```
Should be int right?

Comment (VasurTrekkson) [reply src/main/java/com/lothrazar/storagenetwork/util/RequestBatch.java:37]: ind short for index

Comment (IIpragmaII) [reply src/main/java/com/lothrazar/storagenetwork/util/RequestBatch.java:37]: All right. We can leave it like this then.

**Review by IIpragmaII — APPROVED** [review]

**Comment by GHAFRI** [comment]
Hello, I am very much interested in this subject, can you contact me via discord at: fireflight#9894

**Pull request merged**